### DISCUSSION TITLE
node minio with high cpu

### DISCUSSION THREAD

**Discussion by elton557** [minio/minio#15473] [Show and tell]
Hello, I have a minio cluster with 4 nodes, from time to time 1 node of this cluster has very high cpu consumption and crashes my minio, I can't load files, it just solves by stopping the minio service on this node, what can it be?

![image](https://user-images.githubusercontent.com/52458645/182885154-381803e6-835b-42b4-b631-177de9825ec9.png)

**Comment by harshavardhana** [comment]
You need to tell us more than just this, talk about your setup, minio version etc. we cannot imagine your setup.

**Reply by elton557** [reply]
VERSION:
  RELEASE.2021-03-01T04-20-55Z 

The cluster has 4 nodes

**Reply by ravindk89** [reply]
@elton557 lets not do a drip-feed of information here. More is better. I'll try to give you a little more guidance here, please try to consider adjacent relevant information.

- The configuration of each MinIO host node (CPU, Memory, Storage Controller, Drives)
- The network configuration and adjacent components (load balancers, reverse proxies, etc)
- The type of clients performing ops on the deployment
- The type of workload those clients are performing
- Any logs from the node in question around the time of the increased CPU usage

You are also using a fairly old version of MinIO - well more than a year out of date. There is a distinct possibility your issue may have been resolved in a more recent release. Do you have a UAT or QA/Dev environment in which you can test upgrading to a more recent stable release? It's going to be a big jump so please test _very_ carefully.

**Comment by harshavardhana** [comment]
@elton557 RELEASE.2021-03-01T04-20-55Z please upgrade.
